packages/component/src/createLoadable.js

```diff
@@ -46,8 +47,9 @@ function createLoadable({
   onLoad,
 }) {
   function loadable(loadableConstructor, options = {}) {
-    const ctor = resolveConstructor(loadableConstructor)
     const cache = {}
+    const ctor = resolveConstructor(loadableConstructor)
+    register(ctor);
```

Comment:
you are basically making [imported-components](https://github.com/theKashey/react-imported-component) out of loadable.

Reply:
Sounds like a good thing, knowing you're the author of imported-components :)